lsp-vimscript.el

```diff
@@ -0,0 +1,64 @@
+;;; lsp-vimscript.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 emacs-lsp maintainers
+
+;; Author: emacs-lsp maintainers
+;; Keywords: lsp, vim, vimscript
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP Clients for the VimScript Programming Language.
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-vim nil
+  "LSP support for viml using vim-language-server."
+  :group 'lsp-mode
+  :version "7.1"
```

Comment:
Actually, this server exists since some versions, so I would remove these versions

Reply:
oh, my bad. I had to go out grab fish food else my nephew would cry all the day... back to it!